ElasticSwift.podspec

```diff
@@ -10,10 +10,10 @@ Pod::Spec.new do |s|
   s.author       = { "pksprojects" => "support@pksprojects.com" }
   s.source       = { :git => "https://github.com/pksprojects/ElasticSwift.git", :tag => "v#{s.version}" }
   
-  s.ios.deployment_target = "12.2"
```

Comment:
No need to be so restrictive on iOS, tvOS and watchOS.
I'm testing on iOS 10+
The project is macOS 10.10+, so the same should be for the podspec

Reply:
To take advantages of the new features in swift mainly swift-nio I have based lined ElasticSwift to Swift 5.0. This requires me to lock it to 12.2 else cocoapod  won't let me publish the pod. If its possible not to restrict iOS target, I am willing to go for it.

Reply:
12.2 limitation is not due to swift 5.0 but to swift-nio, and swift-nio is not a swift5.0 feature but a lib mainly for server side dev.
For iOS implementation, at the moment, the 12.2+ limitation is not viable.
Cocoapods understand the swift-nio requirements and push the minimum iOS version to swift-nio minimum version (12.2)

Reply:
This is fixed test with alpha.6